### ISSUE TITLE
Missing schema for GLib

### ISSUE THREAD

**Issue by krzysztofszarek** [jeffreywildman/homebrew-virt-manager#1]
After installation I getting such error: 
```
$ virt-manager --debug
[Sat, 03 Jan 2015 16:45:23 virt-manager 2614] DEBUG (cli:187) Launched with command line: /usr/local/Cellar/virt-manager/1.1.0/libexec/share/virt-manager/virt-manager --debug
[Sat, 03 Jan 2015 16:45:23 virt-manager 2614] DEBUG (virt-manager:148) virt-manager version: 1.1.0
[Sat, 03 Jan 2015 16:45:23 virt-manager 2614] DEBUG (virt-manager:149) virtManager import: <module 'virtManager' from '/usr/local/Cellar/virt-manager/1.1.0/libexec/share/virt-manager/virtManager/__init__.pyc'>
[Sat, 03 Jan 2015 16:45:23 virt-manager 2614] DEBUG (virt-manager:207) GTK version: 3.14.6

(virt-manager:2614): GLib-GIO-ERROR **: Settings schema 'org.virt-manager.virt-manager' is not installed

Trace/BPT trap: 5
```
brew updated, OSX in 10.10.1 version.

**Comment by jeffreywildman** [comment]
Thanks for the feedback!  It looks like remnants from a previous installation of virt-manager on my system masked the fact that neither the glib schema nor hicolor icons were being installed by the virt-manager formula.  I think I've fix this in 2be28dfff02f2d0ee97e91a9b058767082b9433e, please let me know.

**Comment by krzysztofszarek** [comment]
It has fixed my issue. Thanks a lot! :+1:
